### DISCUSSION TITLE
Why don't you add graphql and graphql-java as a topic in About.

### DISCUSSION THREAD

**Discussion by islandshinji** [Netflix/dgs-framework#235] [General]
This framework is not included in the search result of "graphql java".
I thought it would be better to set it to increase the number of users of this framework.
<img width="1562" alt="スクリーンショット 2021-04-05 10 18 04" src="https://user-images.githubusercontent.com/2344406/113527412-6e4e8600-95f8-11eb-8a8e-0a3a8b3850d9.png">

**Comment by paulbakker** [comment]
Thanks for the tip, that was an oversight and I've fixed it now.

**Reply by islandshinji** [reply]
Thank you for addressing this immediately.
